zerver/actions/invites.py

```diff
@@ -362,6 +363,64 @@ def do_get_invites_controlled_by_user(user_profile: UserProfile) -> list[dict[st
     return invites
 
 
+def get_invite_controlled_by_user(prereg_user: PreregistrationUser, user_profile: UserProfile) -> dict[str,Any]:
```

Comment:
Rather than returning `dict[str,Any]`, please define a `TypedDict` that can be shared between this function and `get_multiuse_invite_controlled_by_user`